src/main/java/it/eng/idsa/businesslogic/processor/consumer/websocket/server/HttpWebSocketListenerServer.java

```diff
@@ -0,0 +1,44 @@
+package it.eng.idsa.businesslogic.processor.consumer.websocket.server;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.jetty.websocket.api.Session;
+import org.eclipse.jetty.websocket.api.WebSocketPartialListener;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Example MessagingListenerServer using PartialListener.
+ * @author Antonio Scatoloni
+ */
+public class HttpWebSocketListenerServer implements WebSocketPartialListener {
```

Comment:
- The name of the class should be Http ... or Https ...?  The same question in the other parts of the code.

Reply:
Answer above.